scripts/create_tables.sql

```diff
@@ -1 +1,28 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  specialty CHARACTER VARYING
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  insurance CHARACTER VARYING
+);
+
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  date CHARACTER VARYING,
+  time CHARACTER VARYING,
```

Comment:
Also, not a good idea to name these columns `date` and `time`, since those are also data types in SQL.